### DISCUSSION TITLE
greetings

### DISCUSSION THREAD

**Discussion by juliaochoms42** [ModECI/MDF#214] [General]
hello everyone I am called Tayo Julia Nchoum i am super excited to be part of this great team i am ready to learn know more and also make contributions on this project

**Comment by sanjayankur31** [comment]
Welcome @juliaochoms42 , please take a look at #213 for initial tasks :rocket:

**Comment by juliaochoms42** [comment]
On Tue, 29 Mar 2022 at 08:50 Ankur Sinha ***@***.***> wrote:

> Welcome @juliaochoms42 <https://github.com/juliaochoms42> , please take a
> look at #213 <https://github.com/ModECI/MDF/discussions/213> for initial
> tasks 🚀
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ModECI/MDF/discussions/214#discussioncomment-2459136>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWH76H4LMK5WSN3DJPB24HLVCKY33ANCNFSM5R447WQQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
Hello sir please can you tell me more enlighten me more on the project and
what specific task I’m to carry out
Best Regards,
Tayo Julia

**Reply by sanjayankur31** [reply]
there are a  number of tasks for prospective candidates to work on, please see the other discussions here in the repo.
